composer.json

```diff
@@ -18,17 +18,16 @@
         }
     ],
     "require": {
-        "php":                  "^7.3.0",
-        "doctrine/orm":         "^2.7.4",
+        "php":                  "^7.4.0|| ^8.0",
+        "doctrine/orm":         "^2.7.4|| ^2.8",
         "doctrine/persistence": "^1.3.3 || ^2.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^8.2.0",
-        "infection/infection": "^0.15.2",
+        "infection/infection": "^0.20.2",
         "phpstan/phpstan": "^0.12.52",
         "phpstan/phpstan-phpunit": "^0.12.16",
-        "phpunit/phpunit": "^9.2.6",
-        "roave/backward-compatibility-check": "^4.4",
```

Comment:
Not too fussed about this change. Can we at least run it with an explicit `composer install` in a dedicated dir (`/tmp` also OK)?

Reply:
Alternatively, I could a github action only for 7.4 with a composer require, what do you think?

Reply:
I have added the bc-check job on github action as https://github.com/Roave/BackwardCompatibilityCheck readme advise